src/com/assignments/a7/a7_4/factories/TricycleFactory.java

```diff
@@ -0,0 +1,9 @@
+package com.assignments.a7.a7_4.factories;
+
+import com.assignments.a7.a7_4.implementations.Tricycle;
+
+public class TricycleFactory {
```

Comment:
We can creatre CycleFactory interface having abstract method createCycle which can be implemented by other factories

Reply:
updated